strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/Dependencies.java

```diff
@@ -0,0 +1,80 @@
+package org.carlspring.strongbox.nuget.file;
+
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import javax.xml.bind.annotation.XmlElement;
+
+public class Dependencies
+        implements Serializable
+{
+
+    @XmlElement(
+            name = "dependency",
+            namespace = "http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd"
+    )
+    public List<Dependency> dependencies;
+    @XmlElement(
```

Comment:
Cosmetic: Could you please leave an empty line between the declaration of fields? For example:
```
@SomeAnnotation
Foo foo;

@SomeAnnotation;
Bar bar;
```